src/Stripe/Services/Discounts/StripeDiscountService.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Stripe.Services.Discounts {
+    public class StripeDiscountService :StripeService 
+    {
+        private StripeCustomerService _stripeCustomerService => new StripeCustomerService(ApiKey);
+        private StripeSubscriptionService _stripeSubscriptionService => new StripeSubscriptionService(ApiKey);
+        public StripeDiscountService(string apiKey) : base(apiKey) {}
+
+
+        public virtual StripeDiscountDelete DeleteCustomerDiscount(string customerId, StripeRequestOptions requestOptions = null) 
+        {
+            return _stripeCustomerService.DeleteDiscount(customerId, requestOptions);
+        }
+
+        public virtual StripeDiscountDelete DeleteSubscriptionDiscount(string subscriptionId, StripeRequestOptions requestOptions = null) 
+        {
+            return _stripeSubscriptionService.DeleteDiscount(subscriptionId, requestOptions);
+        }
+
+#if !PORTABLE
```

Comment:
same thing here, just remove the #if and #endif 

Again, work in the same place you did when you created this PR, and then just add, commit, and push to the remote. :)